### ISSUE TITLE
Named parameters does not work can not work as described in readme/contradiction in documentation

### ISSUE THREAD

**Issue by 07pepa** [aws/amazon-redshift-python-driver#71]
## Driver version
at least 2.0.901

## Problem description
readme shows that you can use named arguments (aka keyword arguments (kwargs))
like `cursor.execute(sql, p1=1, p2="hello world")`

however documentation in code says
```
If :data:`paramstyle` is ``named``, the argument should be a dict mapping of parameters.
```

and this is corect because real APi looks like `def execute(self: "Cursor", operation, args=None, stream=None, merge_socket_read=False) -> "Cursor":`

please fix readme since it is incorect or change api of `execute` to allow kwargs 

## Reproduction code
a code from readme siufisses
```python
# named
redshift_connector.paramstyle = 'named'
sql = 'insert into foo(bar, jar) VALUES(:p1, :p2)'
cursor.execute(sql, p1=1, p2="hello world") 
```

**Comment by Brooke-white** [comment]
Hi @07pepa ,

Thank you for bringing this issue to our attention, we will update our documentation.

**Comment by Brooke-white** [comment]
Hi @07pepa ,

We've fixed the documentation in v2.0.902 so I'll be closing this issue. Thanks again for reporting this!
